deps/streamsearch/sbmh.js

```diff
@@ -24,66 +24,76 @@
  * Based heavily on the Streaming Boyer-Moore-Horspool C++ implementation
  * by Hongli Lai at: https://github.com/FooBarWidget/boyer-moore-horspool
  */
-var EventEmitter = require('events').EventEmitter,
-    inherits = require('util').inherits;
+const EventEmitter = require('events').EventEmitter,
+  inherits = require('util').inherits;
 
 function SBMH(needle) {
-  if (typeof needle === 'string')
+  if (typeof needle === 'string') {
     needle = Buffer.from(needle);
-  var i, j, needle_len = needle.length;
+  }
+
+  if ((needle instanceof Buffer) === false) {
```

Comment:
isn't this equivalent to Buffer.isBuffer? shouldn't we pick one and use it everywhere?

Reply:
```suggestion
  if (!Buffer.isBuffer(needle)) {
```